.travis.yml

```diff
@@ -1,43 +1,44 @@
 sudo: required
 language: generic
 
+dist: bionic
+
 services:
 - docker
 
 before_script:
-- curl -sSLf https://get.docker.com | sed s/sleep\ 20/sleep\ 0/g | sudo -E sh
+  - sudo rm -rf /var/lib/apt/lists/*
```

Comment:
See how I extracted this into a shell script. I think it may be easier to maintain that way.

https://github.com/teamserverless/license-check/blob/master/.travis.yml#L14